time timestamp
uint16 sat_id
float64 pseudorange
float64 x
float64 y
float64 z
float64 v_x
float64 v_y
float64 v_z
